compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -123,5 +123,12 @@ class ScalaSettings extends Settings.SettingGroup {
     sys.props("user.dir").split(File.separatorChar).last
   )
 
+  val projectVersion = StringSetting (
+    "-project-version",
+    "project version",
+    "The current version of your project",
+    ""
```

Comment:
If the version isn't specified, it falls back to an empty string. This renders nicely on the site.